### ISSUE TITLE
Angular 2 support

### ISSUE THREAD

**Issue by jbadeau** [tonyhb/tectonic#74]
Hi,

Sounds like a very interesting lib especially for those who cannot use graphql at the moment. My question is, could this lib be split up into something like tectonic.core and techtonic.react so that we could utilize it in other redux env like angular, vue, etc? Is this lib heavy dependent on react?

**Comment by tonyhb** [comment]
Yup, those are the only dependencies on react!  I'll add this to milestone 3.0 to track - it'd be great to do this along with ember integration that other people want. 

Thanks for the heads up!

I'm not too familiar with angular so I may have some questions once the refactoring begins, if that's okay?

**Comment by jbadeau** [comment]
I would me more than willing to help
